core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -1969,4 +1969,14 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
       err.errorReport.statusCode shouldBe Some(StatusCodes.NotImplemented)
     }
   }
+
+  "getSubmissionMethodConfiguration" should "return the method configuration that was used to launch the submission" in withTestDataServices { services =>
```

Comment:
Is it worth testing that this returns the version of the method config used for the submission vs the current version of the method config?

Reply:
Sure, I can make this more robust

Reply:
Updated in c09b2154b12562fd12673b0b000ae01b4311b267